fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -14,6 +14,7 @@ class FMCMainDisplay extends BaseAirliners {
         this.costIndex = undefined;
         this.costIndexSet = undefined;
         this.maxCruiseFL = undefined;
+        this.RECmaxCruiseFL = undefined;
```

Comment:
```suggestion
        this.recMaxCruiseFL = undefined;
```
Would be a little nicer to read 😛 .

Reply:
Fixed!